code/getFFTparameters.m

```diff
@@ -0,0 +1,43 @@
+function [fftParameters] = getFFTparameters(currentFFTconfig,currentTDsampleRate)
+%%
+% Determine FFT parameters from FFTconfig and TD sample rate
+
+%%
+% Decode fftSize
+switch currentFFTconfig.size
+    case 0
+        fftSize = 64;
+    case 1
+        fftSize = 256;
+    case 3
+        fftSize = 1024;
+end
+
+% Determine frequency cutoffs for each FFT bin
+numBins = fftSize/2;
+binWidth = (currentTDsampleRate/2)/numBins;
+
+iCounter = 0;
```

Comment:
If you want a one-liner
```suggestion
fftBins = (0:numBins-1)*binWidth;
```

Reply:
Remove for-loop below.

Reply:
Updated with aa57673e816d91e8410cc36789c1b9a6c46371c3